proto/vultisig/keysign/v1/wasm_execute_contract_payload.proto

```diff
@@ -8,9 +8,14 @@ option go_package = "github.com/vultisig/commondata/go/vultisig/keysign/v1;v1";
 option java_package = "vultisig.keysign.v1";
 option swift_prefix = "VS";
 
+message CosmosCoin {
+  string denom = 1;
+  string amount = 2;
+}
+
 message WasmExecuteContractPayload {
   string sender_address = 1;
   string contract_address = 2;
   string execute_msg = 3;
-  repeated Coin coins = 4;
```

Comment:
usually we can't change the type. but I suppose we didn't release it yet , so will allow it 

But keep in mind , it should not be changed in the future

Reply:
Sure